app/feedme/parsers/enhanced_html_parser.py

```
@@ -0,0 +1,779 @@
+                continue
+        
+        return messages
+
+    def _extract_email_messages(self, soup: BeautifulSoup) -> List[ParsedMessage]:
+        """Extract messages from email thread HTML"""
+        
+        messages = []
+        
+        # Look for email-specific patterns
+        email_blocks = soup.find_all(['div', 'article', 'section'], 
+                                   class_=re.compile(r'(email|mail|message)'))
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)